pkg/extensions/search/cve/cve.go

```diff
@@ -215,7 +216,18 @@ func (cveinfo BaseCveInfo) GetImageListWithCVEFixed(repo, cveID string) ([]commo
 	return fixedTags, nil
 }
 
-func (cveinfo BaseCveInfo) GetCVEListForImage(repo, tag string, pageInput PageInput) (
+func filterCVEList(cveMap map[string]cvemodel.CVE, searchedCVE string, pageFinder *CvePageFinder) {
+	searchedCVE = strings.ToUpper(searchedCVE)
+
```

Comment:
So what happens if searchedCVE is empty string?

Reply:
It will match all the CVEs, because all CVE titles/ids will contain the "" string